tests/unit/cli/admin/test_ceremony.py

```diff
@@ -9,12 +9,21 @@
 class TestCeremony:
     def test_ceremony_with_custom_out(
         self,
+        monkeypatch,
         ceremony_inputs,
+        ceremony_selection,
         client,
         test_context,
         patch_getpass,
         patch_utcnow,
     ):
+
+        # public key selection options
+        monkeypatch.setattr(
+            "repository_service_tuf.cli.admin.helpers._select",
```

Comment:
Make `repository_service_tuf.cli.admin.helpers` as one constant in this file and shorten this to a one line like I did here:
https://github.com/repository-service-tuf/repository-service-tuf-api/blob/6eee4eb83a19e3b7c38171e92ffcb39de1b53979/tests/unit/api/test_bootstrap.py#L14C1-L14C10
it will improve readability and shorten changes.